quinn-proto/src/congestion.rs

```diff
@@ -1,27 +1,65 @@
 //! Logic for controlling the rate at which data is sent
 
-use std::time::{Duration, Instant};
+use crate::connection::paths::RttEstimator;
+use std::fmt::Debug;
+use std::time::Instant;
 
+mod bbr;
+mod bbr_bw_estimation;
+mod bbr_min_max;
 mod cubic;
 mod new_reno;
 
+pub use bbr::{BBRConfig, BBR};
 pub use cubic::{Cubic, CubicConfig};
 pub use new_reno::{NewReno, NewRenoConfig};
 
+/// To help differentiate between packet sizes and packet numbers
+pub type PacketNumber = u64;
```

Comment:
typedefs are not that useful since they still allow wrong types being used. Newtypes (like `struct PacketNumber(u64)`) are more useful. But I think if such a change is applied, it should be done in the whole crate.

Reply:
is that something you'd be interested in? I found it very helpful while developing do know which was which even if it was just in the function def. I had confused the two a few times when developing before I added this.

Reply:
Yeah, I think that would be good to have (but it should likely be a separate PR).

Reply:
okay, I'll remove it from this one

Reply:
fixed